HerPublicWebsite/ExternalServices/GoogleAnalytics/GoogleAnalyticsService.cs

```diff
@@ -19,6 +19,8 @@ public class GoogleAnalyticsService
     private const string EventNameBoilerQuestionViewed = "boiler_question_viewed";
     private const string EventNameQuestionnaireCompleted = "questionnaire_completed";
     private const string EventNameReferralGenerated = "referral_generated";
+    private const string EventNameDigitalAssistance = "digital_assistance";
```

Comment:
We might end up wanting this to be a standard pageview event for consistency with GBIS, or to update GBIS to use the custom `digital_assistance` event. But either way, fine for now.